### PR TITLE
Improvements related to PTVO DIY devices

### PR DESCRIPTION
**Pull request by ptvoinfo**
- Added multi-endpoint version for humidity, pressure, illuminance converters.
- Improved a converter for "ptvo.switch" (dynamic configuration and exposes).

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -3316,6 +3316,60 @@ const converters = {
             return payload;
         },
     },
+    humidity2: {
+        cluster: 'msRelativeHumidity',
+        type: ['attributeReport', 'readResponse'],
+        options: [exposes.options.precision('humidity'), exposes.options.calibration('humidity')],
+        convert: (model, msg, publish, options, meta) => {
```
Comment (Koenkk) [inline converters/fromZigbee.js:3323]: Please integrate these changes into the existing converters.

File: devices/custom_devices_diy.js [hunk 2]
```diff
@@ -280,6 +280,75 @@ const fzLocal = {
     },
 };
 
+function ptvoGetMetaOption(device, key, defaultValue) {
```
Comment (Koenkk) [inline devices/custom_devices_diy.js:283]: Could you use `utils.getMetaValue` instead here?

File: converters/fromZigbee.js (hunk 1 above)
Comment (ptvoinfo) [reply converters/fromZigbee.js:3323]: @Koenkk, I am  agree that if a device supports multiEndpoint, it should always append a suffix. But this update may break integration with HA or other systems for many devices. Therefore, I have added a separate converter.

Comment (Koenkk) [reply converters/fromZigbee.js:3323]: I see, please move it into fzLocal of `custom_devices_diy.js`

Comment (ptvoinfo) [reply converters/fromZigbee.js:3323]: @Koenkk Maybe, add it to fromZigbee.js and start moving to the correct implementation for new devices?

Comment (Koenkk) [reply converters/fromZigbee.js:3323]: I want to go for a diff solution in the future, so lets move it to `custom_devices_diy.js` for now.

**Pull request closed without merging**